# GeoMapper
Creates website for user which has a button to upload a .csv file, which then creates a map for the user which they can download to their computer

uses pandas, flask framework, folium and a library which generates longitude and latitude coordinates depending address given
currently has a strange issue where when user submits their .csv file the website prints "str not callable"
I'm no python wizard but based on the current state I don't see what could generate the issue.
